src/components/mortgage-investment-compare/input-card.js

```diff
@@ -16,18 +17,57 @@ import {
 import { toUSD } from '../../utils/numberFormat'
 
 /**
+ *
  * @param {*} props
- *   @property {number} loanAmt
- *   @property {number} investmentRate
- *   @property {number} inflation
- *   @property {function} onInputChange
- *   @property {Object} option1
- *   @property {Object} option2
- *   @property {Object} shorterOption
- *   @property {Object} longerOption
+ *   @property {number} mustBeGreaterThan - If present, number must be greater
  */
+function ValidatedNumberInput (props) {
+  const [value, setValue] = useState(props.value)
+  const [isValid, setIsValid] = useState(true)
+
+  function onInputChange (event) {
+    const value = event.target.value
+    setValue(value)
+    const parsed = parseFloat(value)
+    if (!value || !_.isNumber(parsed)) {
+      setIsValid(false)
+      return
+    }
+    if (
+      _.isNumber(props.mustBeGreaterThan) &&
+      parsed <= props.mustBeGreaterThan
+    ) {
+      setIsValid(false)
+      return
+    }
+    setIsValid(true)
+    props.onChange(props.name, parsed)
+  }
+
+  return (
+    <Input
```

Comment:
having a `props.mustBeGreaterThan` is great. We can also pass the min attribute to <Input> below and prevent the user from going below 0 with the arrows. In this case you'd pass min={0} when applicable. The user will still be able to change the value without the arrows so that's where `props.mustBeGreaterThan` comes in

https://www.w3schools.com/tags/att_input_min.asp

Reply:
Oh, min is way better, thanks! I'll give it a shot

Reply:
keep in mind they're different @hdavidzhu. min will only work if the user doesn't change the input manually (via typing). `props.mustBeGreaterThan` will likely be triggered most of the time

Reply:
As it turns out, `reactstrap` Input doesn't support `min`, but I renamed the variable for clarity

Reply:
@hdavidzhu it does support it! I tested it before making the suggestion

Reply:
`reactstrap`'s Input is really html's input under the hood

Reply:
`min` seems to work great for me with your new changes!

Reply:
ha so `min` now has 2 uses. The native one, and your custom one